### DISCUSSION TITLE
MacOS compatibility

### DISCUSSION THREAD

**Discussion by Godiesc** [Godiesc/firefox-gx#118] [General]
In MacOS just work this:
1. Base theme
2. Left-sidebar
3. Tab-shapes

The other extra features will not work cause I can't tested or make compatibility for that OS.
You can share your question and issues here if some problem appear.

**Comment by Tintouin** [comment]
## Test of firefoxgx.icon.disabled on macOS
When `firefoxgx.icon.disabled` is **false**:

<img width="355" alt="Capture d’écran 2024-01-07 à 19 37 24" src="https://github.com/Godiesc/firefox-gx/assets/82973239/4fd83268-74ed-404c-b93a-e1281f645231">
<img width="396" alt="Capture d’écran 2024-01-07 à 19 37 36" src="https://github.com/Godiesc/firefox-gx/assets/82973239/534ec381-0197-4380-8df8-53b76782cd17">
<img width="261" alt="Capture d’écran 2024-01-07 à 19 37 45" src="https://github.com/Godiesc/firefox-gx/assets/82973239/4c1ed2af-5961-4070-bbea-ae38cee60cc6">

# <br>

When `firefoxgx.icon.disabled` is **true**:
<img width="359" alt="Capture d’écran 2024-01-07 à 19 41 46" src="https://github.com/Godiesc/firefox-gx/assets/82973239/4860148d-9050-4482-9ac8-0b35efe02886">
<img width="231" alt="Capture d’écran 2024-01-07 à 19 42 09" src="https://github.com/Godiesc/firefox-gx/assets/82973239/714186f6-6b56-476d-acd8-c97118f29fb8">
<img width="383" alt="Capture d’écran 2024-01-07 à 19 41 57" src="https://github.com/Godiesc/firefox-gx/assets/82973239/f7e00674-061c-4b0a-8f34-87db15432f7a">

**Reply by Godiesc** [reply]
It seems to look aceptable, without any item bad positioned. Thanks for the test.

**Comment by NPM26103** [comment]
The OneLine feature doesn’t seem to be working on my setup. Would it be possible for you to check and fix it? I’d really appreciate it. Thank you!

<img width="3020" height="298" alt="CleanShot 2026-03-05 at 11 14 23@2x" src="https://github.com/user-attachments/assets/d0ea5130-33c2-45d8-8818-11b53cb4cf82" />

**Reply by Godiesc** [reply]
hi, sorry but, I can't make 'oneline' configuration compatible because I don't have that OS to test it, you can see in the main github page the compatibility for MacOS:

<img width="465" height="336" alt="image" src="https://github.com/user-attachments/assets/93ae5fe4-42f6-4980-a0fd-004c509e1c33" />

**Comment by c3004-a** [comment]
one line works on the new version
<img width="1512" height="201" alt="Bildschirmfoto 2026-08-06 um 17 08 27" src="https://github.com/user-attachments/assets/865c348a-7654-4cc7-8cca-19bfc153cc06" />

firefox logo being placed in the right left corner is not couses by oneline
